draft-irtf-cfrg-hash-to-curve.md

```diff
@@ -1815,7 +1816,7 @@ Output:
 
 Steps:
 1.  ell = ceil(len_in_bytes / b_in_bytes)
-2.  ABORT if ell > 255
+2.  ABORT if ell > 255 or len_in_bytes > 2^16-1 or len(DST) > 255
```

Comment:
```suggestion
2.  ABORT if ell > 255 or len_in_bytes > 65535 or len(DST) > 255
```

Tiny nit: "65535" is actually 1 character shorter than 2^16-1!

(But I don't feel strongly either way...)

Reply:
That's better =)